### PR TITLE
Advice

### PR DESCRIPTION
**Pull request by thaliabarrera08**
Hi there, just submitting my pull request from "Finishing the Forking Workflow" lesson for Git Hub.

### CODE REVIEW

File: Thalia-barrera-advice.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="css\thalia-barrera.css">
```
Comment (redrambles) [inline Thalia-barrera-advice.html:6]: Let's change `href="css\thalia-barrera.css"` to `href="css/thalia-barrera.css"` (forward slash between 'css' and 'thalia-barrera.css') :)

**Comment by redrambles** [comment]
Hey @thaliabarrera08 !

This is fantastic!! The only thing we need is for you to edit the link to your CSS file from your advice file - right now it has a backslash `\` but we'll need that to be a forward slash `/`

([See comment here](https://github.com/skillcrush/git-practice/pull/722/files#r950289645))

Once you've made that tiny change, save the file, then **add** and **commit** your change and push up your `advice` branch once more, which will automatically update this pull request. 

Then we should be all set to merge in your code! Let us know if you have any questions!

Cheers,
Ann :)

**Comment by thaliabarrera08** [comment]
Ok thank you! I knew something felt off lol Ok will get back to it ASAP

**Comment by thaliabarrera08** [comment]
ok just fixed it , added,committed and pushed it into the repo.
Thanks for all your help!

**Comment by twosavoie** [comment]
Hey @thaliabarrera08,

Looks great! Well done making that change!  

Next, we would like your advice file to be in the advice folder. After moving it there, you'll want to add ../ to the front of the file path to your css file (Since your advice file is in a folder now, you'll need the file path to go up a level to the root of the project to find the css folder with the css file.) 

Then compare the way the advice file name is spelled to how it's spelled in the link in the index.html file. You'll want it to be exactly the same, including capitalization. 

When you're finished, you'll run a slightly different git command (`git add -A`), commit, then push, your changes will join your pull request here.

Let us know if you need any help!

xoxo,
Lisa

**Comment by redrambles** [comment]
Hey @thaliabarrera08 !

How is it going with this? Do you need a hand? Don't be shy to let us know!

Cheers,
Ann :)

**Comment by twosavoie** [comment]
Hey @thaliabarrera08,

We haven't heard from you in a while so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

xoxo,
Lisa

**Comment by thaliabarrera08** [comment]
OOH! I forgot about fixing the rest of it! sorry about that. I got too
excited and was skipping ahead to the next classes.

**Comment by twosavoie** [comment]
Completely understandable @thaliabarrera08! Let us know if you have any questions on the next steps. 

xoxo,
Lisa

**Comment by twosavoie** [comment]
Hey @thaliabarrera08,

We haven't heard from you in a while, so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

xoxo,
Lisa

**Pull request closed without merging**